pkg/crc/machine/vfkit/constants.go

```diff
@@ -5,14 +5,13 @@ package vfkit
 
 import (
 	"fmt"
-	"runtime"
 )
 
 const (
 	VfkitVersion = "0.0.1"
+	VfkitCommand = "vfkit-universal"
```

Comment:
We added `-${arch}` suffixes to our macos binaries as otherwise there would have been a name conflict. Isn't this a good opportunity to rename them back to just `vfkit` and `crc-admin-helper-darwin`? We don't add the arch on the other aplatforms.

Reply:
yes, makes sense to do this instead of adding `-universal` for these